### PR TITLE
Replace translatable html attribute with translate which is valid

### PR DESCRIPTION
**Pull request by KKoukiou**
translatable is not a valid html attribute and was creating a lot of
noise in html validators.

### CODE REVIEW

**Comment by andreasn** [comment]
I don't feel I have enough domain-knowledge about translations to review this.
Maybe @mvollmer would be a good reviewer. I think he did the initial work on translations in Cockpit.

File: pkg/users/index.html [hunk 1]
```diff
@@ -433,7 +433,7 @@ <h4 class="modal-title" translatable="yes">Add public key</h4>
       <div class="pf-c-alert__icon">
           <i class="fa fa-info-circle" aria-hidden="true"></i>
       </div>
-      <h4 translatable class="pf-c-alert__title">The user must log out and log back in to fully change roles.</h4>
```
Comment (marusak) [inline pkg/users/index.html]: This seems like it never worked and this string was missing in translations. (And weirdly we had `translatable` as string to be translated instead)

File: tools/test-static-code [hunk 2]
```diff
@@ -50,6 +50,11 @@ if out=$(find src/ pkg/ -name '*.js' -o -name '*.jsx' | xargs grep -n -E "(gette
     echo "$out" >&2
     echo "not ok 3 js-translatable-strings"
     fail=1
+elif out=$(grep -r 'translatable=["'\'']yes' pkg/ doc/); then
```
Comment (marusak) [inline tools/test-static-code]: I would not limit to to `=yes`, I think that `grep -r 'translatable='` would be enough, but just nitpick :)

File: pkg/users/index.html (hunk 1 above)
Comment (garrett) [reply pkg/users/index.html]: Some HTML attributes duplicate themselves if they're included without a value. For example: `disabled` becomes `disabled="disabled"`. (Perhaps something like that happened here?)

File: tools/test-static-code (hunk 2 above)
Comment (KKoukiou) [reply tools/test-static-code]: Since I fixed all occurrences let's keep it like this so that there is some consistency regarding how we use it.

**Pull request merged**